config/default.schema.json

```diff
@@ -487,6 +487,11 @@
 			"type": "boolean",
 			"default": true,
 			"description": "Enables button to new Nuxt School Admin beta-version page"
+		},
+		"FEATURE_NEW_LOGIN_ENABLED": {
```

Comment:
what does new login mean? is there a more descriptive variant available like FEATURE_OAUTH_LOGIN_ENABLED?

Reply:
OK, Done